pkg/restapi/issuer/operation/operations.go

```diff
@@ -652,7 +670,6 @@ func (o *Operation) composeAndIssueCredentialHandler(rw http.ResponseWriter, req
 
 		return
 	}
-
```

Comment:
You should remove these unrelated formatting changes (removal of line breaks) in order to keep your PR focused on the one thing it says it does

Reply:
fixed!